doctorapp/src/App.vue

```diff
@@ -4,4 +4,87 @@
 
 <script setup lang="ts"></script>
 
-<style scoped></style>
\ No newline at end of file
+<style scoped></style>
+
+
+<style>
```

Comment:
Why are there 2 style tags here? It's best that you opted for a non-scoped style tag, as the styling needs to go to every component
```suggestion
<style>
```

Reply:
in this case I would even prioritise a `style.css` file to have a cleaner `App.vue` and not have 85% of it be global styles